Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-08-11
Initial Package Version: 1.6.7
Upstream Status:         Not submitted
Origin:                  RedHat Development CVS
Description:             Fixes GCC4 compilation errors

$LastChangedBy$
$Date$


--- guile-1.6.7/libguile/guile.c.ltdl	2005-03-02 16:31:46.000000000 +0100
+++ guile-1.6.7/libguile/guile.c	2005-03-02 16:32:07.000000000 +0100
@@ -53,6 +53,7 @@
 #endif
 
 #ifdef DYNAMIC_LINKING
+#include <ltdl.h>
 #include "guile-ltdl.h"
 #endif
 
@@ -88,7 +89,7 @@
 {
 #ifdef DYNAMIC_LINKING
   /* libtool automagically inserts this variable into your executable... */
-  extern const scm_lt_dlsymlist lt_preloaded_symbols[];
+  extern const lt_dlsymlist lt_preloaded_symbols[];
   scm_lt_dlpreload_default (lt_preloaded_symbols);
 #endif
   scm_boot_guile (argc, argv, inner_main, 0);
